src/components/core/DpBulkEditHeader.stories.tsx

```diff
@@ -0,0 +1,89 @@
+import type { Meta, StoryObj } from '@storybook/vue'
+import DpBulkEditHeader from './DpBulkEditHeader.vue'
+import DpButton from "~/components/DpButton/DpButton.vue";
+
+const meta: Meta<typeof DpBulkEditHeader> = {
+    component: DpBulkEditHeader,
+    title: 'Components/BulkEditHeader',
+    render: (args) => ({
+        components: {
+            DpButton,
+            DpBulkEditHeader
+        },
+        computed: {
+            selectedItemsCount () {
+                return args.options.filter(checked => checked.checked == true)
+            }
+        },
+        setup() {
+            const selectAll = () => {
+                for (let element of args.options) {
+                    element.checked = true
+                }
+            }
+
+            const resetSelection = () => {
+                const selectAll = document.getElementById('selectAll')
+                for (const element of args.options) {
+                    element.checked = false
+                }
+                selectAll.checked = false
+            }
+
+            return { args, resetSelection, selectAll }
+        },
+        template: `
+          <div>
+              <dp-bulk-edit-header
+                  v-if="selectedItemsCount.length > 0"
+                  :selected-items-text="selectedItemsCount.length + ' ' + 'Elements selected'"
+                  @reset-selection="resetSelection"
+                  v-bind="args">
+                <dp-button
+                    text="Delete"
+                    icon="fa-trash"
```

Comment:
There is no icon `fa-trash` in DpIcon. This is why the `warning` icon is rendered as a fallback. The `delete` icon is more appropriate here.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/615/commits/aff06c048674a339549873b0989a933239e33357

Reply:
Please look at the docs in `http://localhost:6006/?path=/docs/components-icon--docs#delete` (local storybook) how it is used.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/615/commits/1aa637355f94cc7c8f030c88ec64db5319bacc71